blocks/teaser-list/teaser-list.js

```diff
@@ -67,8 +67,7 @@ export default async function decorate(block) {
   const pageList = await lookupPages(pathnames);
   if (pageList.length) {
     pageList.forEach((row) => {
-      // If the URL was not in the index, it is curated. Let's get the content differently
-      if (row.title === undefined) setRowDetails(row, blockCopy);
+      setRowDetails(row, blockCopy);
```

Comment:
Does this allow for any of eyebrow, title, description or link title to be overriden? I don't think eybrow can be overridden and linktext should only be overriden if its inner text !== its href.

Reply:
Elaborating on Anthony's comment:
- Teaser-list doesn't use eyebrows, so we're OK there ✔️ 
- When only headline + link is present, then description will auto populate ✔️ 
- When only description + link is present, then headline will auto populate ✔️ 
- When more than 1 paragraph is authored, only the 1st one shows 🆗 (I don't think teaser-list is expected to show more than 1).
- when inner text == its href, then "Learn more about" should be the default link text ❌